workers.tf

```diff
@@ -95,7 +95,7 @@ resource "aws_autoscaling_group" "workers" {
           var.worker_groups[count.index],
           "autoscaling_enabled",
           local.workers_group_defaults["autoscaling_enabled"],
-        ) == 1 ? "enabled" : "disabled"}"
```

Comment:
there is no need to do  == "true". In expression `local.some_var ? this : that`,  local.some_var must be either bool or string that can be converted to bool, e.g. "true" or "false"